src/excalidraw-app/index.tsx

```diff
@@ -285,9 +249,10 @@ function ExcalidrawWrapper() {
     elements: readonly ExcalidrawElement[],
     appState: AppState,
   ) => {
-    saveDebounced(elements, appState);
     if (collabAPI?.isCollaborating) {
```

Comment:
Can you add a comment saying that we intentionally want to only save when outside of collaboration